src/attach-shadow.js

```diff
@@ -279,7 +280,12 @@ ShadyRoot.prototype.removeEventListener = function(type, fn, optionsOrCapture) {
 }
 
 ShadyRoot.prototype.getElementById = function(id) {
-  return this.querySelector(`#${id}`);
+  let result = mutation.query(this, function(n) {
```

Comment:
I'd like to get @sorvell's opinion on if this is a good idea to use mutations here.